hack/release/make-release-tag.sh

```diff
@@ -0,0 +1,84 @@
+#! /usr/bin/env bash
+
+# make-release-tag.sh: This script assumes that you are on a branch and have
+# otherwise prepared the release. It rewrites the image tag in the operator's
+# deployment manifest, then creates a tag with a message containing the shortlog
+# from the previous version. Note: The Envoy image is not managed by this script.
+
+readonly PROGNAME=$(basename "$0")
+readonly OLDVERS="$1"
+readonly NEWVERS="$2"
+
+if [ -z "$OLDVERS" ] || [ -z "$NEWVERS" ]; then
+    printf "Usage: %s OLDVERS NEWVERS\n" "$PROGNAME"
+    exit 1
+fi
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+readonly IMG="docker.io/projectcontour/contour:$NEWVERS"
+
+# If you are running this script, there's a good chance you switched
+# branches, do ensure the vendor cache is current.
+go mod vendor
+
+if [ -n "$(git tag --list "$NEWVERS")" ]; then
+    printf "%s: tag '%s' already exists\n" "$PROGNAME" "$NEWVERS"
+    exit 1
+fi
+
+# Wrap sed to deal with GNU and BSD sed flags.
+run::sed() {
+    local -r vers="$(sed --version < /dev/null 2>&1 | grep -q GNU && echo gnu || echo bsd)"
+    case "$vers" in
+        gnu) sed -i "$@" ;;
+        *) sed -i '' "$@" ;;
+    esac
+}
+
+# Update the Docker image tags in the operator's deployment manifest.
+# The version might be main or OLDVERS depending on whether we are
+# tagging from the release branch or from main.
+run::sed \
+    "-es|docker.io/projectcontour/contour:main|$IMG|" \
+    "-es|docker.io/projectcontour/contour:$OLDVERS|$IMG|" \
+    "-es|docker.io/projectcontour/contour-operator:main|$IMG|" \
```

Comment:
This needs a new var above since it would replace the `contour-operator` image with `contour` on lines 46 & 47.

Reply:
@stevesloka I don't believe so since we plan on synchronizing contour/operator releases. The operator should have the same release version as Contour. For example v1alpha1 of the operator will be v1.11 when released on 12/4 with contour v1.11.

Reply:
@danehans looks like the issue is that L47 and L48 replace not just the version/tag for the operator, but replace the entire `contour-operator:<version>` string with `contour:<version>` which is not desirable.

Reply:
@skriss commit `001327e` fixes the issue. Thanks for the explanation.